// Copyright 2019 The MediaPipe Authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto2";

package mediapipe.autoflip;

import "mediapipe/framework/calculator.proto";

message LocalizationToRegionCalculatorOptions {
  extend mediapipe.CalculatorOptions {
    optional LocalizationToRegionCalculatorOptions ext = 284226721;
  }

  // Output standard autoflip signals only (Human, Pet, Car, etc) and apply
  // standard autoflip labels.
  optional bool output_standard_signals = 1 [default = true];
  // Output all signals (regardless of label) and set autoflip label as
  // 'Object'.  Can be combined with output_standard_signals giving each
  // detection a 'object' label and a autoflip sepcific label.
  optional bool output_all_signals = 2 [default = false];
}
